src/aics-image-viewer/components/App/types.ts

```diff
@@ -29,16 +29,26 @@ type ControlNames =
 /** Show/hide different elements of the UI */
 export type ControlVisibilityFlags = { [K in ControlNames]: boolean };
 
+export type MultisceneUrls = { scenes: (string | string[])[] };
+
 export interface AppProps {
   // FIRST WAY TO GET DATA INTO THE VIEWER: pass in volume data directly
+
   // rawData has a "dtype" which is expected to be "uint8", a "shape":[c,z,y,x] and a "buffer" which is a DataView
   rawData?: RawArrayData;
   // rawDims is a small amount of metadata (e.g. dimensions and channel names) to be converted internally to an ImageInfo
   rawDims?: RawArrayInfo;
 
   // SECOND WAY TO GET DATA INTO THE VIEWER: (if `rawData`/`rawDims` isn't present) pass in URL(s) to fetch volume data
-  imageUrl: string | string[];
-  parentImageUrl?: string | string[];
+
+  /**
+   * URL(s) from which to fetch the image. You can pass a `string` to load from a single data source, or get fancier:
+   * - Pass an array of strings to assemble a single volume with all sources' channels, in order.
+   * - Pass an object with a key `scenes: (string | string[])[]` to load multiple volumes as a *multi-scene collection*.
+   *   Each string or string array within the `scenes` array is treated as a single volume with one or more sources.
+   */
+  imageUrl: string | string[] | MultisceneUrls;
+  parentImageUrl?: string | string[] | MultisceneUrls;
```

Comment:
is it impossible to just have these accept MultisceneUrls?

Reply:
Same thing - would be a breaking change that we'd have to propagate to NBVV, CFE, etc. Or are you asking to include `string | string[]` in the `MultisceneUrls` type?

Reply:
I understand that here it's a compatibility problem.  PArt of me wants to say we can break older versions and accept that nbvv and cfe need updates.  Or, since we are still sort of prototyping, I would really even consider a new named prop called multisceneImageUrl : MultisceneUrls but I'm not psyched about these unions of types overloading the same variable names.